meson.build

```diff
@@ -5,12 +5,18 @@ project(
     version : run_command(find_program('python3'), '-m', 'setuptools_scm', check: true).stdout().strip(),
 )
 
-# Specify the compile-time limb size. Exactly one of `32`, `64` or `NATIVE`
+# Specify the compile-time limb size. Exactly one of `32`, `64` or, `NATIVE`
 add_project_arguments(
     '-DCOMPILER_LIMB_SIZE=NATIVE',
     language: ['cpp', 'c'],
 )
 
+# Build with(out) nanobind leak warnings enabled. Exactly one of `1` or `0`.
+add_project_arguments(
+    '-DNANOBIND_LEAK_WARNINGS=0',
```

Comment:
Should it perhaps be enabled by default on debug builds?

Reply:
That's a great idea, thanks!